Gems/PhysX/Code/Editor/ColliderCapsuleMode.h

```diff
@@ -11,13 +11,15 @@
 #include <Editor/Source/ComponentModes/PhysXSubComponentModeBase.h>
 #include <AzToolsFramework/ComponentModes/BoxViewportEdit.h>
 #include <AzFramework/Entity/EntityDebugDisplayBus.h>
+#include <AzToolsFramework/ComponentModes/CapsuleViewportEdit.h>
 
 namespace PhysX
 {
     /// Sub component mode for modifying the height and radius on a capsule collider.
     class ColliderCapsuleMode
         : public PhysXSubComponentModeBase
         , private AzFramework::EntityDebugDisplayEventBus::Handler
+        , private AzToolsFramework::CapsuleViewportEdit
```

Comment:
Is there a reason why we call it Mode here while we call it Edit in the framework?

Reply:
It is a component mode here, but the framework code is for a class which can either be used in a component mode or not.